workflow/executor/resource.go

```diff
@@ -50,6 +50,10 @@ func (we *WorkflowExecutor) ExecResource(action string, manifestPath string) (st
 		args = append(args, string(buff))
 	}
 
+	if we.Template.Resource.IgnoreNotFound {
```

Comment:
We should also make sure that we only allow `ignoreNotFound` when `action == "get"` and any other relevant action.

Reply:
I've checked, and the flag is only valid for get and delete.

Reply:
`for CMD in $(kubectl | grep -e '^  [^ ]' | awk {print\ \$1}); do echo $CMD; kubectl $CMD --help | grep ignore-not-found; done`

Reply:
Great, thanks for this!